<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qcoba</class>
 <widget class="QMainWindow" name="qcoba">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qcoba</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="grpSerial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>121</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial Port (8-N-1)</string>
    </property>
    <widget class="QComboBox" name="cmbBaud">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cmbPort">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ttyUSB0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyUSB2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyATK0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyATK1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyATK2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyACM0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyACM1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ttyACM2</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btnOpen">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpDatanumber">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLineEdit" name="txtVnumb">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtV1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtV2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtV3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblData1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>V #</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblData1_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>V1</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblData1_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>V2</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblData1_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>V3</string>
     </property>
    </widget>
   </widget>
   <widget class="QwtPlot" name="pltData">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>511</width>
      <height>201</height>
     </rect>
    </property>
    <property name="canvasBackground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
    <property name="autoReplot">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtCommData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
